cg/cli/workflow/tomte/base.py

```diff
@@ -0,0 +1,19 @@
+"""CLI support to create config and/or start TOMTE."""
+
+import logging
+
+import click
+
+from cg.constants.constants import MetaApis
+from cg.meta.workflow.analysis import AnalysisAPI
+from cg.meta.workflow.tomte import TomteAnalysisAPI
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group(invoke_without_command=True)
+@click.pass_context
+def tomte(context: click.Context) -> None:
+    """gms/tomte analysis workflow."""
```

Comment:
Not sure if relevant, but as TOMTE is used in capital letters in the description of the module I think is better to keep it unified
```suggestion
    """GMS/TOMTE analysis workflow."""
```